source/content/create-custom-upstream.md

```diff
@@ -147,6 +147,14 @@ To avoid incompatibilities, you must track Pantheon's corresponding upstream rep
 
     </Tab>
 
+    <Tab title="Drupal 9" id="d91">
+
+    ```bash{promptUser: user}
+    git remote add pantheon-drupal-9 git://github.com/pantheon-upstreams/drupal-project.git
```

Comment:
I believe the official D9 repo is https://github.com/pantheon-systems/drupal-project

Reply:
`pantheon-upstreams/drupal-project` is correct.  `pantheon-systems/drupal-project` is the source repository that generates it.